### DISCUSSION TITLE
Number of connections a lot more than number of channels

### DISCUSSION THREAD

**Discussion by thayhoang** [rabbitmq/rabbitmq-server#3482] [Other]
We're publishing messages to Rabbitmq using Mqtt protocol. On our production environment, I noticed the number of connections is a lot more than number of channels. As my understanding, to connect to an AMQP cluster, you need at least 1 channel.  What's happening here? Does my cluster have any problem?

![image](https://user-images.githubusercontent.com/17960508/134336379-4a1b3d1d-117a-463e-b498-e9663dfcaca2.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
A connection can exist without any open channels. We cannot comment on "what's happening here" but your applications can fail to open a connection or run into a [channel-level exception](https://www.rabbitmq.com/channels.html#error-handling) and not notice/recover from that.

You can see what channels are open on a given connection by navigating to its page.

**Reply by thayhoang** [reply]
Thanks @michaelklishin 
"A connection can exist without any open channels" 
@michaelklishin Is it true in case of Mqtt protocol? As I know, mqtt is a stateful protocol so should it have at least 1 channel?
